source/user/manual/material/uniaxialMaterials/HystereticSM.rst

```diff
@@ -120,17 +120,137 @@ The following input format is compatible with Hysteretic material. Note that in
    * - defoLimitStatesDCRMaxAbs
      - MaximumAbsolute-deformation DCR on User-Defined Limit States
      - array with the ratio of the MAXIMUM strain to each of the envelope strain points (max(emaxP,abs(emaxN))/els)
-
    * - forceLimitStatesDCR
      - force DCR on User-Defined Limit States
      - array with the ratio of the CURRENT stress/force to each of the user-defined force limit states
-
-
    * - allData
      - All relevant Data
      - all relevant data at current step (mom1p, rot1p, mom2p, rot2p, mom3p, rot3p, mom4p, rot4p, mom5p, rot5p, mom6p, rot6p, mom7p, rot7p, mom1n, rot1n, mom2n, rot2n, mom3n, rot3n, mom4n, rot4n, mom5n, rot5n, mom6n, rot6n, mom7n, rot7n, pinchX, pinchY, damfc1, damfc2, beta, CrotMax, CrotMin, CrotPu, CrotNu, CenergyD, CloadIndicator, Cstress, Cstrain, Ttangent)
 
+------------
+
+**Example Input:**
+
+ops.uniaxialMaterial('HystereticSM', 99, '-posEnv', 2772.0, 0.01, 3104.6, 0.02, 1663.2, 0.04, 1663.2, 0.06, 277.2, 0.08, 200.0, 0.1, 0, 0.12, '-negEnv', -2772.0, -0.01, -3104.6, -0.02, -1663.2, -0.04, '-pinch', 1, 1,'-damage', 0.1, 0.01, '-beta', 0,'-defoLimitStates', 0.01, -0.01, 0.02, -0.02, '-forceLimitStates', 2772.0, -2772.0, 3104.6, -3104.6,'printInput')
+
+uniaxialMaterial HystereticSM  99  -posEnv  2772.0  0.01  3104.6  0.02  1663.2  0.04  1663.2  0.06  277.2  0.08  200.0  0.1  0  0.12  -negEnv  -2772.0  -0.01  -3104.6  -0.02  -1663.2  -0.04  -pinch  1  1 -damage  0.1  0.01 -beta 0 -defoLimitStates 0.01 -0.01 0.02 -0.02 -forceLimitStates 2772.0 -2772.0 3104.6 -3104.6 -printInput
+
+Open or download Jupyter notebook with example of HystereticSM material, used generate the figures `here: <https://github.com/silviamazzoni/OpenSeesDocumentation/blob/master/source/user/manual/material/uniaxialMaterials/examples/HystereticSM_materialTest.ipynb>`_
+------------

```

Comment:
Open or download Jupyter notebook with example of HystereticSM material, used generate the figures `here: <https://github.com/silviamazzoni/OpenSeesDocumentation/blob/master/source/user/manual/material/uniaxialMaterials/examples/HystereticSM_materialTest.ipynb>`_
------------------------------------------------------------------------------------------------------------------------